### PR TITLE
KAN-4 Web Login re-design

### PR DESCRIPTION
**Pull request by XimenaPulidoGutierrez**
###  Description
The design of the login has been updated

### Solution
I created a second version of the login, added strings, and checked which colors worked with the interface.
### Notes
- n/a
### Tickets
[TICKET] (https://one-sm.atlassian.net/browse/KAN-4)
### Screenshots / Screencasts
image

### CODE REVIEW

File: src/pages/auth/LoginV2.tsx [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+          </Form.Item>
+          <div className="text-right mb-4">
+            <Link className=" font-semibold" to={ResetPasswordRoute}>
+              {Strings.forgotPassword}
+            </Link>
+          </div>
+          <Form.Item className="text-center">
+            <Button
+              loading={isLoading}
+              className="px-12 py-4 text-lg mt-4"
```
Comment (imadiaz) [inline src/pages/auth/LoginV2.tsx:141]: can we make the button fill width to the form content, add a more radius, and remove the background shape?

Comment (XimenaPulidoGutierrez) [reply src/pages/auth/LoginV2.tsx:141]: What did you mean with "add a more radius"?

Comment (imadiaz) [reply src/pages/auth/LoginV2.tsx:141]: Also make sure that the design is responsive

Comment (XimenaPulidoGutierrez) [reply src/pages/auth/LoginV2.tsx]: Yes! I already made sure of that. I have already made the requested corrections.

**Review by imadiaz — APPROVED** [review]

**Review by Isaacjbar — APPROVED** [review]
lgtm

**Pull request merged**